certification/internal/shell/base_on_ubi_test.go

```diff
@@ -3,42 +3,70 @@ package shell
 import (
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/cli"
+	log "github.com/sirupsen/logrus"
 )
 
+// podmanEngine is a package-level variable. In some tests, we
+// override it with a "happy path" engine, that returns good data.
+// In the unhappy path, we override it with an engine that returns
+// nothing but errors.
```

Comment:
This comment is epic and very clear! Nothing to change here, just wanted to thank you for adding this. Resolve this thread whenever 😆